lib/app/views/guided.erb

```diff
@@ -75,7 +75,7 @@
           No actively running launch sequences found for iss <%=params[:iss]%>.
           Please ensure that the EHR launch test is actively running before attempting to launch Inferno from the EHR.
         <% elsif error_code == 'no_state' %>
-          Inferno has detected an issue with the SMART launch. No actively running launch sequences found with a state of <%=params[:state]%>.
+          Inferno has detected an issue with the SMART launch. No actively running launch sequences found with a state parameter of '<%=params[:state]%>'.
```

Comment:
it would be nice to have the same logic here that lets the user no state parameter was given.

e.g. here I gave no state parameter but Inferno was saying one was found with value of ''

![Screen Shot 2020-06-26 at 9 53 01 AM](https://user-images.githubusercontent.com/41651655/85864733-e8f99380-b792-11ea-80f5-9007e65f3bc7.png)

Reply:
Valid suggestion, but I'm going to leave this as-is.